doc/changes.rst

```diff
@@ -56,6 +56,23 @@ Varnish Cache Next (2021-03-15)
 * All shard ``Error`` and ``Notice`` messages now use the unified
   prefix ``vmod_directors: shard %s``.
 
+* The ``VCL_SUB`` data type is now supported for VMODs to save
+  references to subroutines to be called later using
+  ``VRT_call()``. This will fail the VCL for calls from a wrong
```

Comment:
Better:  "Calls from a wrong context will fail (..."

Reply:
yes, thank you, fixed with next force push